test/integration/test-namedPlaceholders.js

```diff
@@ -36,5 +37,13 @@ test('Test namedPlaceholder as command parameter', {
       assert.equal(rows[0].result, 1);
       c.end();
     });
+  },
+  'Disabled in pool config, enable query command': () => {
+    const c = createPool({ namedPlaceholders: false });
+    c.query({ sql: query, namedPlaceholders: false }, values, (err, rows) => {
```

Comment:
This tests namedPlaceholders disabled in both initial config and local override, is that intended?

Reply:
I wanted to see, if the tests in CI pass as well, because they should fail and didn't on my system. So looks like they don't actually get verified, or am I wrong?

Reply:
oh, ok, I'll try to investigate

Reply:
at first I thought this might be a problem with the pool, but it seems to broken one test above as well

Reply:
Looks like utest is finishing before any commands even start, and it says there is no planned support for async

I guess none of the tests in that file ever worked. One easy way to fix is to just cange to simple run + assert similar to other tests ( this is starting to become messy, I'd like to refactor all tests one day and migrate so some modern test runner like jest/mocha or ava )

Reply:
So should I remove/fix the test again or do you want to merge this anyways?

Reply:
If you have time would be great if you can rewrite that test. If not let me know and I'll merge pr and fix the test myself

Reply:
Don't have the time to fix the whole test setup right now, but I could help with jest in the future.
But would be nice if you could merge this pr until then ^^